x-pack/plugins/infra/public/pages/metrics/hosts/hooks/use_control_panels_url_state.ts

```diff
@@ -140,6 +179,16 @@ const PanelRT = rt.type({
       fieldName: rt.string,
       title: rt.union([rt.string, rt.undefined]),
       selectedOptions: rt.array(rt.string),
+      helpMessage: rt.intersection([
```

Comment:
To keep it consistent with the title component we can have a message with a mandatory `text` property and the `link` property is optional - if there is a link provided to the message the link properties are required